dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -1578,6 +1585,25 @@ public void cleanup(){
 		getWritableNullProperties().clear();
 	}
 
+	@Override
+	public ManifestInfo getManifestInfo() {
+		String type = null;
+
+		if (Host.class.isInstance(this.getClass()) || this.isHost()) {
+			type = PusheableAsset.SITE.getType();
+		} else  {
+			type =  PusheableAsset.CONTENTLET.getType();
+		}
+
+		return new ManifestInfo(map(
+				"object type", type,
```

Comment:
yeah this should be constants ManifestInfo.OBJECT_TYPE

Reply:
was removed